### DISCUSSION TITLE
upload Zigbee firmware using Tasmota 9.2.0 failed

### DISCUSSION THREAD

**Discussion by manrot05** [arendst/Tasmota#10729] [Q&A] [answered]
Hi
Problem: **I can not flash the zigbee chip with the firmware fw_SonoffZigbeeBridge_ezsp**

Here the steps I did: (on a windows10 machine)

- I flashed Sonoff bridge with tasmota-zbbridge.bin using tasmotizer 1.2. to version 9.2.0 
- connected bridge to my WiFi, gave it fixed IP-address
- UI is showing "Tasmota 9.2.0 by Theo Arends"
- downloaded ncp-uart-sw_6.7.6_115200.ota file from Github
- I choose from menu 'Firmware Upgrade'
- Upload started ... => Upload transfer ... => after a while I get error msg: **Upload Failed** Upload aborted

When I look in console screen I see these messages:
23:33:07 UPL: File ncp-uart-sw_6.7.6_115200.ota ...
23:33:11 UPL: Transfer 190272 bytes
23:33:11 UPL: Successful 190272 bytes

What did I do wrong? Can anyone help me?

grt Manfred

**Comment by sfromis** [comment, marked as answer]
Ensure that you have `Module 75` activated to upload EZSP firmare, "Sonoff ZbBridge Module" at the top of the web UI.

**Reply by manrot05** [reply]
ok I am a newbie and just followed some instructions found on youtube. Were the same action was done and It magically worked 

how do I activate 'module 75'

**Reply by sfromis** [reply]
Do *not* assume that some old YouTube video is a good guide when we have the actual documentation online for the Tasmota on the [ZbBridge](https://zigbee.blakadder.com/Sonoff_ZBBridge.html). You can set/check the module using the Module command in the console, or via configure module.

**Reply by manrot05** [reply]
Ok, I should have linked the Youtube video in my response, since it is only 1 day old.
[Sonoff Zigbee Bridge | How to flash with Tasmota and setup in Home Assistant with ZHA](https://www.youtube.com/watch?v=SDynPbCGvS0)

I checked your link  to blackadder ZbBridge and I can not see any difference from what I did, or what the YT video showed
Flash ESP8266 was no problem.

Flash Zigbee module ends with failure in my case.
- Go to Firmware Upgrade and next to “Upgrade by file upload” use the Choose File button and select Zigbee module firmware you downloaded (ncp-uart-sw_6.7.8_115200.ota).
- Click on Start upgrade, be patient and wait for a few minutes until flashing is complete. Once it is done, Tasmota will reboot. If the flash was successful the console will show Zigbee2Tasmota starting:
- .... ZIG: Zigbee started

It says: "Follow further instructions depending on your chosen method."
and then additional instructions are given, but only after flashing the firmware.
And that part ended in error in my case. I do not see zigbee started

**Regarding module 75** I checked via configure module and it was already selected, but I reselected it and zbbridge was restarted, => repeated the steps ended in the same failure

I added some screenshots, maybe I am still missing something.
<img width="387" alt="2021-01-28 11_51_27-Tasmota - Information" src="https://user-images.githubusercontent.com/77198812/106135717-96d58200-6168-11eb-913b-306ac1df5fa9.png">
<img width="284" alt="2021-01-28 12_59_43-Tasmota - Configure Module" src="https://user-images.githubusercontent.com/77198812/106135824-b8cf0480-6168-11eb-99a2-473a58671814.png">
<img width="944" alt="2021-01-28 13_00_35-Tasmota - Console" src="https://user-images.githubusercontent.com/77198812/106136402-8bcf2180-6169-11eb-9a19-5baa6ad37bb6.png">
<img width="298" alt="2021-01-28 13_04_03-Tasmota - Console" src="https://user-images.githubusercontent.com/77198812/106136255-56c2cf00-6169-11eb-8c06-a6ca3d09d001.png">

**Reply by manrot05** [reply]
I have no idea how to solve this problem
I tried with 4 different kind of flash firmware versions
8.5.1
9.1
9.2.0.3
9.2.0

but still upload failed 

followed the steps on https://zigbee.blakadder.com/Sonoff_ZBBridge.html
and added the intructions 'module 75'

please help?
